src/basilisp/core.lpy

```diff
@@ -4374,7 +4374,24 @@
   this function."
   [& paths]
   (doseq [path (seq paths)]
-    (load-file (str path ".lpy"))))
+    (let [path-rel (if (.startswith path "/")
+                     path
+                     (let [nsname (.-name *ns*)
+                           dot-last (.rfind nsname ".")
+                           path-parent (-> (subs nsname 0 (if (= dot-last -1) (count nsname) dot-last))
+                                           (.replace "." "/"))]
+                       (str "/" path-parent "/" path)))
+          path-rel (.replace path-rel "-" "_")
```

Comment:
```suggestion
    (let [path-rel (if (.startswith path "/")
                     path
                     (let [path-parent (-> (name *ns*)
                                           (.rsplit "." 1)
                                           (first))]
                       (str "/" path-parent "/" path)))
          path-rel (.replace path-rel "-" "_")
```

This seems a bit simpler and requires less manual substring work. Does it achieve the same effect?

Reply:
Updated.